src/Components/MovieListPage/MovieListPage.js

```diff
@@ -1,63 +1,41 @@
-import React, {useState, useEffect, useCallback, useRef} from "react";
+import React, {useState, useEffect, useCallback, useRef, createContext} from "react";
 import styles from './MovieListPage.module.scss';
 import GenreSelector from '../GenreSelector/GenreSelector';
 import SortControl from '../SortControl/SortControl';
 import MoviesFound from '../MoviesFound/MoviesFound';
 import MovieTile from '../MovieTile/MovieTile';
 import Logo from '../Logo/Logo';
-import Header from '../Header/Header';
-import MovieDetails from "../MovieDetails/MovieDetails";
-import SwitchComponents from "../SwitchComponents/SwitchComponents";
+import {Outlet} from "react-router-dom";
 import {GenreListArray, SortControlArray} from '../../data';
 
 const url = 'http://localhost:4000/movies?';
+const SearchContext = createContext({
+    onSearch: null,
+    isLoading: null,
+});
 
 const MovieListPage = () => {
```

Comment:
I'd recommend to create folder src/pages
and move the MovieListPage and ErrorPage there

In that case you will have reusable components in one place, and pages in the other place